Jenkinsfile

```diff
@@ -1,107 +1 @@
-// This Jenkinsfile's main purpose is to show a real-world-ish example
-// of what Pipeline config syntax actually looks like.
```

Comment:
Obsolete as there are plenty of other examples of Declarative including in tests, on jenkins.io, etc.